sisl/io/siesta/out.py

```diff
@@ -239,7 +239,7 @@ def next_geom():
         return next_geom()[1]
 
     @sile_fh_open()
-    def read_force(self, last=True, all=False):
+    def read_force(self, last=True, all=False, total=False, maxF=False):
```

Comment:
I would prefer `max` rather than `maxF`, it is implicit is about forces :)

Reply:
Yes, I did it not to overwrite the `max()` builtin function, but if we don't ever plan to use it, I guess it's safe to use max.

Reply:
I thought so ;)
One can easily do `from numpy import amax as ...` or `_max = max` outside the function. So I think we could bypass this :)